pkg/cloudprovider/providers/aws/tags.go

```diff
@@ -137,14 +137,14 @@ func (t *awsTagging) hasClusterTag(tags []*ec2.Tag) bool {
 	clusterTagKey := t.clusterTagKey()
 	for _, tag := range tags {
 		tagKey := aws.StringValue(tag.Key)
+                // Check if this is a newer-style cluster tag before checking if legacy tag value matches ClusterID
```

Comment:
Looks like your indentation is off on this comment here

Reply:
Yep. I see that. Will change.